google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -515,6 +563,30 @@ private Builder() {
                               .build())
                       .build());
 
+      long numOfChannels = 2 * Runtime.getRuntime().availableProcessors();
```

Comment:
I think you should extract this to a static package protected method and use it here and in defaultGrpcTransportProviderBuilder

```
static int getDefaultChannelPoolSize() {
   return 2 * Runtime.getRuntime().availableProcessors();
}
public static defaultGrpcTransportProviderBuilder() {
return BigtableStubSettings.defaultGrpcTransportProviderBuilder()
  .setPoolSize(getDefaultChannelPoolSize())
  .setMaxInboundMessageSize(268435456);
}
```